01_fib/joel00wood/main.go

```diff
@@ -0,0 +1,41 @@
+package main
+
+import (
+	"fmt"
+	"math"
+)
+
+func fibCalc(n float64) float64 {
+	switch n {
+	case 0:
```

Comment:
nitpick: stylistically I'd lean more towards either having an early out conditional:

```go
if n <= 1 {
  return n
}
```

or combining your first two cases:

```go
switch n {
    case 0, 1:
        return n
/* ... etc ... */
```

Reply:
Seems reasonable to me, I think I did attempt option two initially but for whatever reason it didn't work so I stuck with the expanded approach

Reply:
Updated